### PR TITLE
CPR-367 Add cert to docker file and add properties

### PR DESCRIPTION
**Pull request by H-Iwanejko**

### CODE REVIEW

File: src/main/resources/application-postgres.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+spring.datasource.url=jdbc:postgresql://${DATABASE_ENDPOINT}/${DATABASE_NAME}?sslmode=verify-full
+spring.datasource.username=${DATABASE_USERNAME}
+spring.datasource.password=${DATABASE_PASSWORD}
```
Comment (markreesmoj) [inline src/main/resources/application-postgres.properties:3]: There is an existing `spring.datasource.url` in `application.yml` (where everything is specified in the URL).
We can split it into separate properties like this by all means - it's easier to read

Comment (markreesmoj) [reply src/main/resources/application-postgres.properties:3]: As it stands this file won't be loaded so these properties won't be set. We can either set a the spring profile to `postgres` in all environments, or move this into `application.yml`

If we do end up keeping it in a separate file, we should make it `yml` so all our properties are in the same format

**Review by markreesmoj — APPROVED** [review]
👍

File: src/main/resources/application-postgres.properties (hunk 1 above)
Comment (H-Iwanejko) [reply src/main/resources/application-postgres.properties]: Update 👍 We can move out into seperate files in subsequent updates

**Pull request merged**